lib/broadway.ex

```diff
@@ -600,6 +600,22 @@ defmodule Broadway do
        `:transformer` callback will cause the whole producer to terminate,
        possibly leaving unacknowledged messages along the way.
 
+    * `:rate_limiting` - Optional. A list of options to enable and configure
```

Comment:
Hrm, sorry for the nitpick at this moment, but should this be called `rate_limiter` instead?

Reply:
I prefer `rate_limiting` because it describes a behaviour instead of an implementer of that behaviour. No strong preference though.

Reply:
@msaraiva agrees with you. So this is good to go!